NCR-Corporation/ncr-retail-demo
# #4 Fixing inability to update user profiles

## Description

This fixes the ability to update a user profile.
Part of the problem came from:
1 - not having the correct roles - NEP_IDENTITY_ADMINISTRATOR (I think)
2 - not updating with the correct account.

In this scenario, a "user" does not update their own account. The "system" can update accounts. This is seen in the `sendAuthenticatedRequest` vs `sendRequest`.

`sendAuthenticatedRequest` is for when a user wants to do user specific things with their `AccessToken`
`sendRequest` is the system using hmac

## Screenshots/Screencasts

![Screenshot 2021-03-11 093627](https://user-images.githubusercontent.com/73493569/110803502-42d0b980-824d-11eb-84d1-a7e62d181c61.jpg)


## Testing

1. Login as a user
2. Update profile account
3. Confirm Success

## Ticket

Ticket Number
